extension-manifest-v2/src/classes/ConfData.js

```diff
@@ -107,6 +107,7 @@ class ConfData {
 			_initProperty('current_theme', 'default');
 			_initProperty('enable_ad_block', false);
 			_initProperty('enable_anti_tracking', false);
+			_initProperty('enable_autoconsent', true);
```

Comment:
Would it make sense to have the same logic as adblocking and anti-tracking? Starting with being disabled and then activating after onboarding?

Reply:
Good catch. Makes perfect sense

Reply:
This is not that simple. We have to cover a case, where users update from an earlier version. Then, it has to be set to `true` (there is no other option, which we could migrate). I did not find a better place to initialize if that happens, so if it is `true` by default and disabled if onboarding triggers, every case is covered.

Reply:
I see. I think the canonical way in Ghostery to handle migrations is to add code here:

https://github.com/ghostery/ghostery-extension/blob/f7cc668efd375cae83fc9b60741040731dd7d78e/extension-manifest-v2/src/background.js#L1287

```
function initializeVersioning() {
  ...
  if (globals.JUST_UPGRADED) {
    ...
    if (utils.semverCompare(PREVIOUS_EXTENSION_VERSION, '8.8.0') < 0) {
	conf.enable_autoconsent = true;
    }
  }
}
```

That would run once for users before 8.8.0. And initializeVersioning also runs before initializeGhosteryModules, which triggers the onboarding.

Reply:
Ok, I will change to use your suggestion.

Reply:
Thanks, one thing that I just realized. 8.8.0 might need to be changed, since we pushed that already in Ghostery Dawn. Instead of 8.8.0 it should be the next release version. (I thought 8.7.6 is the latest, but it is 8.8.0. Sorry).

Reply:
I used `8.9.0`.